features/step_definitions/api_request.rb

```diff
@@ -58,11 +58,23 @@
 end
 
 Given("A first tier immigration case") do
+  @assessment_data[:level_of_help] = "controlled"
   @proceeding_type_data = { "proceeding_types": [{ ccms_code: CFEConstants::IMMIGRATION_PROCEEDING_TYPE_CCMS_CODE, client_involvement_type: "A" }] }
 end
 
 Given("A first tier asylum case") do
```

Comment:
IMHO we should have the case type in the test name - otherwise us devs can't work out if the case is controlled or certificated without coming and looking in this bit of the code

Reply:
I don't think you qujte understood my comment. I thought the cucumber step was going to be called 'Given a first tier controlled immigration case' so that we couldn't accidentally mix 'certificated' and 'first tier' (and the same for certificated upper tribunal)